deploy/script/katello-service

```diff
@@ -20,13 +20,9 @@ if [ $EUID -ne 0 ]; then
   exit 1
 fi
 
-if [ -x /etc/init.d/tomcat6 -o -x /lib/systemd/system/tomcat6.service ]; then
-  TOMCAT="tomcat6"
-fi
+# Reversed order so qpid comes before tomcat - BZ1147263
+SERVICES=$(/bin/ls -r /etc/katello/services.d)
```

Comment:
What about using file ordering to more clearly define and control the service ordering? Similar to how Foreman seeding works (https://github.com/theforeman/foreman/tree/develop/db/seeds.d) or how Apache controls configuration ordering?